6MI0800226/week2.cpp

```diff
@@ -0,0 +1,160 @@
+#include <iostream>
+#include <fstream>
+#include <assert.h>
+ 
+const int BUFF_SIZE = 1024;
+void saveResultsInFile(int num1, int num2, const char location[BUFF_SIZE])
+{
+    std::ofstream file(location);
+ 
+    if (!file.is_open())
+        return;
+ 
+    file << (num1 + num2) << std::endl << (num1 * num2);
+}
+int findDiffInFile(const char location[BUFF_SIZE])
+{
```

Comment:
Contrary to the [single responsibility principle](https://en.wikipedia.org/wiki/Single-responsibility_principle) this function is doing many things at once. If it was me - I'd create a function to read me a number from an existing `std::ifstream` (e.g. something like `int readInt(std::ifstream& input)`). Then I'll reuse the said function twice